### ISSUE TITLE
Please add site http://www.4ksw.com

### ISSUE THREAD

**Issue by XXuen** [dteviot/WebToEpub#1093]
URL of a certain story: http://www.4ksw.com/49/49383/

Did you try using the Default Parser for the site?
Ans: Yes i did tried but it pop out as giberrish as in lots of question mark here and there (its a chinese novel)

![image](https://github.com/dteviot/WebToEpub/assets/115346001/5d7e2eef-97a1-43f9-b5f7-c0236eae6cfc)

### If the Default Parser did not work, if you have developer skills, did you try writing a new parser?
I don't have enough skill set to do it nor friends to help

I suspect is something to do with languse but im really not sure about it.

thanks for taking the time to look through

**Comment by dteviot** [comment]
@XXuen 

Test versions for Firefox and Chrome have been uploaded to https://drive.google.com/drive/folders/1B_X2WcsaI_eg9yA-5bHJb8VeTZGKExl8?usp=sharing. Pick the one suitable for you, follow the "How to install from Source (for people who are not developers)" instructions at https://github.com/dteviot/WebToEpub/tree/ExperimentalTabMode#user-content-how-to-install-from-source-for-people-who-are-not-developers and let me know how it goes.
Tested with:
* http://www.4ksw.com/49/49383/, chapters 1 &amp; 2

Caution, I don't know Chinese, so I can't guarantee I've got this right.

For my notes: 27 minutes work
